src/util/createMetaModule.js

```diff
@@ -0,0 +1,60 @@
+module.exports = function createMetaModule(mapFunction, moduleOptions){
+
+  moduleOptions = moduleOptions || {};
+  moduleOptions.infoJson = moduleOptions.infoJson || 'info.json';
+
+  function MetaModule(options, UI) {
+
+    var defaults = require('./getDefaults.js')(require(moduleOptions.infoJson));
```

Comment:
I'm seeing:

```
  Load colorbar opaque meta module
    Error: Cannot find module 'info.json'
```

I think this path is not getting set properly...

Reply:
I think since require statements are run at compile time I think it is trying to find an info file in the util function directory. It think it should rather get the whole file from the module. Like
Util:
```js
moduleOptions.infoJson = moduleOptions.infoJson || {};
```

Module
```js
require('createMetaModule')(mapFunction, require('./info.json'));
```

This should mostly resolve the error.